mountpoint-s3/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Unreleased
 
+## v1.17.0 (May 13, 2025)
```

Comment:
This is in the future! 😄

Reply:
This was a sad way to find out that Monday isn't a bank holiday. 😄